client/src/store/historyStore/model/filtering.js

```diff
@@ -115,6 +115,31 @@ const validFilters = {
     update_time_lt: compare("update_time", "lt", toDate),
 };
 
+/** Valid filter fields and handlers to return filterSettings. */
+const settingsFilters = {
+    "create_time=": compare("create_time", "le", toDate),
```

Comment:
This is very similar to `validFilters` above, we should probably consolidate the two instead of duplicating them.

Reply:
I tried doing that but since `validFilters` is used to parse the `filterText` into something like `"deleted":"false"`, but I made `settingsFilters` to parse it into `"deleted=":"false"`. This is an example:

![image](https://user-images.githubusercontent.com/78516064/165964521-07a0f122-44ed-4c89-8d29-35bf75b71b84.png)

Is it ok if we just replaced the original `validFilters` with the `settingsFilters` I made? I can try that out.

Reply:
Never mind, that (`settingsFilters`) turned out to be so redundant. I could have just used `validFilters`, pushing the changes now..